digid_eherkenning/templates/digid_eherkenning/post_binding.html

```diff
@@ -1,13 +1,16 @@
 {% load static %}
 <html>
     <head>
-        <title>Ga naar DigID</title>
+        <title>U wordt doorgestuurd</title>
+        <link rel="stylesheet" type="text/css" href='{% static "login_submit.css" %}'>
```

Comment:
```suggestion
        <link rel="stylesheet" type="text/css" href='{% static "digid_eherkenning/login_submit.css" %}'>
```
please properly namespace assets

Reply:
Agreed, felt a bit hacky